todo_app/app.py

```diff
@@ -8,7 +11,17 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', headers=['id', 'status', 'title'], results=get_items())
```

Comment:
This is quite a generic solution and for me is a bit overkill - it would be simpler to have the relevant headers hardcoded on the page and access the individual item elements explicitly rather than looping over every value. It's quite likely that our items will end up with fields that don't want to be displayed (or require more thought in how we display them)